.github/workflows/ci.yml

```
@@ -45,3 +45,13 @@ jobs:
               mkdocs build
             env: |
               FULL_BUILD=true
+      
+      - name: Lint the static site
+        uses: anishathalye/proof-html@v2
```

I will disable it for now until I fully cleanup the reports locally via incremental changes

```suggestion
        uses: anishathalye/proof-html@v2
        # Disabled
        if: ${{ false }}
```